src/cli/commands/beacon.ts

```diff
@@ -36,11 +41,28 @@ export class BeaconNodeCommand implements CliCommand {
           address: options.depositContract
         },
         provider: await this.getProvider(options)
+      },
+      rpc: {
+        apis: this.setupRPC(options.rpc.split(","))
       }
     });
     await node.start();
   }
 
+  private setupRPC(args: string[]): IApiConstructor[] {
+    let apis: IApiConstructor[];
+    if (args.includes("beacon")) {
+      apis.push(BeaconApi);
+    }
+    if (args.includes("validator")) {
```

Comment:
I think we can probably iterate all exported apis and fetch namespace. 

Something like this: https://github.com/ChainSafe/lodestar/blob/master/src/cli/index.ts#L12